src/NuGet.Core/NuGet.Packaging/PackageArchiveReader.cs

```diff
@@ -282,6 +282,27 @@ public override Task<bool> IsSignedAsync(CancellationToken token)
             return Task.FromResult(isSigned);
         }
 
+        public override Task<bool> IsZip64Async(CancellationToken token)
+        {
+            token.ThrowIfCancellationRequested();
+
+            if (ZipReadStream == null)
+            {
+                throw new SignatureException(Strings.SignedPackageUnableToAccessSignature);
+            }
+
+            var isZip64 = false;
+
+#if IS_DESKTOP
+            using (var reader = new BinaryReader(ZipReadStream))
+            {
+                isZip64 = SignedPackageArchiveUtility.IsZip64(reader);
+            }
+#endif
+
+            return Task.FromResult(isZip64);
```

Comment:
It seems kind of misleading that this will always return false for NETCore, should it do something else?  I'm just worried about something depending on this and later being broken if it returned the correct result.